cmd/get/bom.go

```diff
@@ -0,0 +1,67 @@
+package get
+
+import (
+	"log"
+
+	mctl "github.com/metal-toolbox/mctl/cmd"
+	"github.com/metal-toolbox/mctl/internal/app"
+	"github.com/spf13/cobra"
+	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
+)
+
+type getBomInfoByBmcMacAddressFlags struct {
+	aocMacAddr string
+	bmcMacAddr string
+}
+
+func (gb *getBomInfoByBmcMacAddressFlags) hasAOCMacAddr() bool {
+	return gb.aocMacAddr != ""
+}
+
+func (gb *getBomInfoByBmcMacAddressFlags) hasBMCMacAddr() bool {
+	return gb.bmcMacAddr != ""
+}
+
+var (
+	flagsGetBomByMacAddress *getBomInfoByBmcMacAddressFlags
+)
+
+var getBomInfoByMacAddress = &cobra.Command{
+	Use:   "bom",
+	Short: "Get bom object by AOC or BMC Addr",
+	Run: func(cmd *cobra.Command, args []string) {
+		if !flagsGetBomByMacAddress.hasAOCMacAddr() && !flagsGetBomByMacAddress.hasBMCMacAddr() {
+			log.Fatalf("--aoc-mac and --bmc-mac not set")
+		}
+
+		if flagsGetBomByMacAddress.hasAOCMacAddr() && flagsGetBomByMacAddress.hasBMCMacAddr() {
+			log.Fatalf("--aoc-mac and --bmc-mac should not set simutansly")
```

Comment:
Spelling on `simultaneously`. Also, defer this to Cobra: https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsMutuallyExclusive